system_tests/logging_.py

```diff
@@ -175,3 +176,25 @@ def test_create_sink_bigquery_dataset(self):
         sink.create()
         self.to_delete.append(sink)
         self.assertTrue(sink.exists())
+
+    def test_create_sink_pubsub_topic(self):
+        from gcloud import pubsub
+
+        # Create the destination topic, and set up the IAM policy to allow
+        # Cloud Logging to write into it.
+        pubsub_client = pubsub.Client()
+        topic = pubsub_client.topic(TOPIC_NAME)
+        topic.create()
+        self.to_delete.append(topic)
+        policy = topic.get_iam_policy()
+        policy.owners.add(policy.group('cloud-logs@google.com'))
+        topic.set_iam_policy(policy)
+
+        TOPIC_URI = 'pubsub.googleapis.com/%s' % (topic.full_name,)
```

Comment:
This PR is the last one left before I propose merging the `logging-api` branch to master.  It seems to me like you have a question for the logging API team, rather than an actual objection to code here: I don't know what the resolution would be -- given that the storage destination prefix doesn't fit the pattern, why add the bit of cleverness you propose to this test?